app/views/quotes/index.html.erb

```diff
@@ -9,7 +9,7 @@
       </footer>
     </blockquote>
     <div class="created">
-      <%= time_ago_in_words(quote.created_at) %> ago
+        <%= time_ago_in_words(quote.created_at || Time.now) %> ago    
```

Comment:
This feels like a workaround, are there entries in the database that have a null entry?